### ISSUE TITLE
Create a "whats happening" page

### ISSUE THREAD

**Issue by nmunro** [spirit-of-christmas/spectrum#130]
Create a "whats happening" page, this might be a bullitin board style view on events that are happening in the next week.

**Comment by BhosalePriyanka** [comment]
Where should we have that button link for "whats happening" page ? Is it on main page on navrbar or somewhere else. 
On what "whats happening" page should we keep same format like we have before. or you are expecting other.?

Navbar - 
<img width="1420" alt="Navbar" src="https://github.com/spirit-of-christmas/spectrum/assets/52287282/2d6b1e55-b8e2-4dea-8ce4-f01e6baff8b0">

Format-
<img width="1380" alt="format" src="https://github.com/spirit-of-christmas/spectrum/assets/52287282/9b822a85-1892-4fef-9406-0f355d263467">

**Comment by nmunro** [comment]
This would be an entirely new view, instead of all events, it would show only the events happening in the next 7 days.

I have no initial ideas on how this view would look (if you have some ideas please show me some mock ups).

I'm thinking a new entry to the navbar but also a link in the main search view to immediately browse the most imminent events.

**Comment by BhosalePriyanka** [comment]
We can keep it in table form like 
<img width="887" alt="Table" src="https://github.com/spirit-of-christmas/spectrum/assets/52287282/f5322123-761a-4bf4-9148-6f3f13b74172">

**Comment by nmunro** [comment]
I mean, we could, I was wondering if we could make something a little more ambitious and visually pleasing.

**Comment by BhosalePriyanka** [comment]
<img width="574" alt="Board" src="https://github.com/spirit-of-christmas/spectrum/assets/52287282/80303e39-feef-45b2-9834-5a9332f5d72e">

**Comment by nmunro** [comment]
That might be quite cool, yes

**Comment by BhosalePriyanka** [comment]
<img width="1411" alt="1" src="https://github.com/spirit-of-christmas/spectrum/assets/52287282/6169a965-c3c9-4a27-984d-4ce049365959">


Are you expecting to have "What Happeing" details on event page means on this page?
If this page then it has already this format, then ?

**Comment by BhosalePriyanka** [comment]
Should i work on this?

**Comment by nmunro** [comment]
No, it will require back end work that I don't have time to do right now. I'm finishing of #118 and setting up the legal requirements for third party cookies.
